etc/delphix-platform/upgrade/download-latest-image

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+URL="http://ops.jenkins.delphix.com"
+JOB="job/devops-gate/job/master/job/appliance-build/job/master/job/post-push"
+
+function die() {
+	echo "$(basename "$0"): $*" >&2
+	exit 1
+}
+
+function usage() {
+	echo "$(basename "$0"): $*" >&2
+	echo "Usage: $(basename "$0") [-f] <variant>"
+	exit 2
+}
+
+opt_f=false
+while getopts ':f' c; do
+	case "$c" in
+	f) eval "opt_$c=true" ;;
+	*) usage "illegal option -- $OPTARG" ;;
+	esac
+done
+shift $((OPTIND - 1))
+
+[[ $# -gt 1 ]] && usage "too many arguments specified"
+[[ $# -eq 0 ]] && usage "too few arguments specified"
+
+S3_PREFIX="$(curl --silent "$URL/$JOB/lastSuccessfulBuild/artifact/latest")"
```

Comment:
Any particular reasons for not just getting it from `s3://snapshot-de-images/builds/jenkins-ops/devops-gate/master/appliance-build/master/post-push/latest`?

Reply:
That'll probably work; I can make that change.

S3 is "eventually consistent" so I think we could end up downloading a slightly-older-than-latest image, but I don't think that'll be a practical problem.